docs/api.golden_record.rst

```diff
@@ -110,7 +109,7 @@ HTTP 200 code response
 .. code-block:: rst
 
    [
-    {"nl_question": "question", "sql": "sql_query", "db":"db_alias", "organzation":"dataherald"},
+   {"id": "id", "question": "question", "sql_query":"sql", db_alias: "database alias"},
    ]
```

Comment:
is the `id` related to k2-server?

Reply:
You might need it for deleting the golden records? @DishenWang2023